godot_project/utils/workspace_utils.gd

```diff
@@ -712,6 +715,90 @@ static func _is_invalid_mol_file(in_path: String) -> bool:
 	return false
 
 
+static func _is_msep_workspace(in_path: String) -> bool:
+	return in_path.get_extension() == "msep1"
+
+
+## Imports an MSEP workspace from disk, inside the current workspace.
+## The structure hierarchy from the imported file is preserved and added under
+## the current active group.
+static func _import_msep_workspace(out_workspace_context: WorkspaceContext, path: String,
+						placement: ImportFileDialog.Placement, snapshot_name: String) -> void:
+	var imported_workspace: Workspace = ResourceLoader.load(path, "", ResourceLoader.CACHE_MODE_IGNORE)
+	if not imported_workspace:
+		Editor_Utils.get_editor().prompt_error_msg("Cannot load file: " + path)
+		return
+	
+	# Clear selection in other contexts
+	var other_contexts: Array[StructureContext] = out_workspace_context.get_structure_contexts_with_selection()
+	for context: StructureContext in other_contexts:
+		context.clear_selection()
```

Comment:
nit: you can replace all this block for
`out_workspace_context.clear_all_selection()`